test/boilerplate_suggestors/advanced_props_and_state_class_migrator_test.dart

```diff
@@ -12,16 +12,27 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-import 'dart:convert';
-
 import 'package:over_react_codemod/src/boilerplate_suggestors/advanced_props_and_state_class_migrator.dart';
 import 'package:over_react_codemod/src/boilerplate_suggestors/boilerplate_utilities.dart';
 import 'package:test/test.dart';
 
 import '../util.dart';
-import 'boilerplate_utilities_test.dart';
 
 void main() {
+  group('AdvancedPropsAndStateClassMigrator', () {
+    AdvancedPropsAndStateClassMigratorTestHelper();
+
+    group('with --treat-all-components-as-private flag', () {
+      AdvancedPropsAndStateClassMigratorTestHelper(
+        shouldTreatAllComponentsAsPrivate: true,
+      );
+    });
+  });
+}
+
+void AdvancedPropsAndStateClassMigratorTestHelper({
```

Comment:
#nit only classes should be named using `UpperCamelCase`

```suggestion
void advancedPropsAndStateClassMigratorTestHelper({
```